src/sentry/relay/projectconfig_cache/redis.py

```diff
@@ -1,57 +1,41 @@
 from sentry.relay.projectconfig_cache.base import ProjectConfigCache
-from sentry.utils import json
-from sentry.utils.redis import get_dynamic_cluster_from_options, validate_dynamic_cluster
+from sentry.utils import json, redis
+from sentry.utils.redis import validate_dynamic_cluster
 
 REDIS_CACHE_TIMEOUT = 3600  # 1 hr
 
 
 class RedisProjectConfigCache(ProjectConfigCache):
     def __init__(self, **options):
-        self.is_redis_cluster, self.cluster, options = get_dynamic_cluster_from_options(
-            "SENTRY_RELAY_PROJECTCONFIG_CACHE_OPTIONS", options
-        )
+        cluster_key = options.get("cluster", "default")
+        self.cluster = redis.redis_clusters.get(cluster_key)
```

Comment:
This should work in prod, we have `rc-ingest` configured there.
https://github.com/getsentry/getsentry/blob/master/getsentry/conf/settings/prod.py#L669